src/main/kotlin/org/cascadebot/cascadebot/data/objects/GuildSettingsCore.kt

```diff
@@ -36,6 +37,8 @@ class GuildSettingsCore {
     @Setting
     var helpShowAllModules = false
 
+    var writeMode = false
```

Comment:
This should not be a separate field on its own, should use the main guild-data's one